### PR TITLE
Add algo Bug and Feature Templates

### PR DESCRIPTION
**Pull request by jo-tm**
as I added Risk issue template, we need also a Bug and a Feature reqquests, otherwise the menu is empty.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (netrome) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Perhaps add a custom bug report label

**Review by netrome — APPROVED** [review]
Just suggesting introducing labels for the types.

**Comment by igorsyl** [comment]
I love machine-readable labels! I will ingest these using GitHub GraphQL straight into GPT-4 for parsing and information extraction.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jo-tm) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: there is already a `bug` label by default.

Comment (netrome) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: Nice :+1:

**Review by netrome — APPROVED** [review]

**Review by jferrant — APPROVED** [review]

**Pull request merged**